docs/index.md

```diff
@@ -10,6 +10,7 @@ Get started with the docs below.
 installation.md
 quickstart.md
 metrics.md
+tutorials.md
```

Comment:
(index.md is basically the root of the documentation tree, so Sphinx will start recursively collecting pages linked from here)